packages/world/src/interfaces/IStoreRegistrationSystem.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.8.0;
 
 /* Autogenerated file. Do not edit manually. */
 
-import { Schema, IStoreHook } from "./../modules/core/implementations/StoreRegistrationSystem.sol";
+import { Schema, IStoreHook } from "../modules/core/implementations/StoreRegistrationSystem.sol";
```

Comment:
@dk1a Same question here as above with import path

Reply:
here the reason is different from `IWorldKernel` - I always want to prefix paths with `./` because local unix paths can start without it (and thus `path.join` etc can also reduce a path to e.g. `src/interfaces/...`), but local solidity paths must always start with `./`. I can check the string for `../` and conditionally not prefix it, but that's unnecessary complexity